proteus/WaveTools.py

```diff
@@ -0,0 +1,241 @@
+"""Tools for working with water waves
+The primary objective of this module is to provide solutions (exact and
+
+approximate) for the free surface deformation and subsurface velocity
+components of water waves. These can be used as boundary conditions,
+wave generation sources, and validation solutions for numerical wave
+codes.
+
+"""
+from math import pi,tanh,sqrt,exp,log,sin,cos,cosh,sinh
+import numpy as np
+from matplotlib  import pyplot
+
+
+
+def sigma(omega,omega0):
+    """sigma function for JONSWAP spectrum
+    """
+    sigmaReturn = omega.copy()
+    sigmaReturn[:] = 0.07
```

Comment:
if omega is an NumPy array, this is slightly more Pythonic: `sigmaReturn = np.empty_like(omega); sigmaReturn[:] = 0.07`